### DISCUSSION TITLE
When I try to compile hpipm, I get a fatal C1083, what should I do?

### DISCUSSION THREAD

**Discussion by rogueyan5** [giaf/hpipm#129] [Q&A]
It looks like the project is missing header files like blasfeo_target.h and blasfeo_timing.h. How do I fix this, should I change the CMakeLists?
```
D:\code\Third-party Libraries\hpipm\auxiliary\../include/hpipm_timing.h(41,10): fatal error C1083: 无法打开包括文件: “blasfeo_timing.h”: No such file or directory [D:\code\Third-party Libraries\hpipm\build\hpipm.
```
Sincerely looking forward to receving your reply!

**Comment by giaf** [comment]
These header files are part of the BLASFEO project, that is a dependency of HPIPM.
Make sure that you give the correct BLASFEO installation path in the HPIPM cmake by setting BLASFEO_PATH to the installation folder in your machine.
You can do this by either modifying the default value in CMakeLists (i.e. replace the default value `/opt/blasfeo`, that is the default value on linux machines, with your actual BLASFEO installation directory), or by passing it to cmake with
```
cmake .. -DBLASFEO_PATH=/path/to/your/blasfeo/installation/folder
```
